src/rely/reporters/JunitReporter.re

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */;
+let testResultToTestCase = (testResult: Reporter.testResult) => {
+  let time =
+    switch (testResult.duration) {
+    | Some(Seconds(s)) => s
+    | None => 0.
+    };
+
+  switch (testResult.testStatus) {
+  | Passed =>
+    Junit.Testcase.pass(
+      ~name=testResult.title,
+      ~classname=testResult.fullName,
+      ~time,
+    )
+  | Skipped =>
+    Junit.Testcase.skipped(
+      ~name=testResult.title,
+      ~classname=testResult.fullName,
+      ~time,
+    )
+  | Failed(message, location_opt, stack) =>
```

Comment:
I think we might want to play with making the last argument ```String.concat("\n\n", [message, stack])```, (and maybe include the filename and line number from the location in the message), but I'm not sure. Do you mind playing around with what this looks like in Azure so that we can ensure we can best communicate the relevant information?

Reply:
Tests running with this change, added some failures to really test it: https://strid.visualstudio.com/OSS/_build/results?buildId=398

Reply:
It seems like Azure DevOps is picking up the testcases classname in a bad way. I think that is a issue on Azure DevOps side and not this implementation.